airgun/entities/host.py

```diff
@@ -84,6 +84,13 @@ def search(self, value):
         view = self.navigate_to(self, 'All')
         return view.search(value)
 
+    def new_ui_button(self):
+        """Click New UI button and return the browser URL"""
+        view = self.navigate_to(self, 'All')
+        view.new_ui_button.click()
+        view.wait_displayed()
+        return self.browser.url
```

Comment:
browser url can be fetch using session object inside test module, no need to return from this function.
```suggestion
```